src/advert/specific_hide.txt

```diff
@@ -566,6 +567,7 @@ investor.id##.row > .show-desktop > div
 senimovie.info##.rsABlock
 duwitmu.com##.section_sticky_ads_bottom
 semprot.com##.semprotnenenmontok_2
+mangaku.vip##.separator
```

Comment:
i don't see ads. did you mean `manganime.in##.separator` ?

Reply:
Fixed ! thank you for the review